File: src/Projects/Android/016-TextGeneratorSystemNetLib/Server/src/main/java/org/csystem/app/generator/text/server/configuration/datetime/DateTimeFormatterConfig.java
package org.csystem.app.generator.text.server.configuration.datetime;

import org.springframework.context.annotation.Bean;
import org.springframework.context.annotation.Configuration;

import java.time.format.DateTimeFormatter;

@Configuration
public class DateTimeFormatterConfig {
    @Bean
    public DateTimeFormatter createFormatter()
    {
        return DateTimeFormatter.ISO_DATE_TIME;
    }
}

File: src/Projects/Android/ChatSystem/Server/src/main/java/org/csystem/app/chat/server/configuration/datetime/DateTimeFormatterConfig.kt
package org.csystem.app.chat.server.configuration.datetime

import org.springframework.context.annotation.Bean
import org.springframework.context.annotation.Configuration
import java.time.format.DateTimeFormatter

@Configuration
open class DateTimeFormatterConfig {
    @Bean
    open fun createFormatter(): DateTimeFormatter
    {
        return DateTimeFormatter.ISO_DATE_TIME
    }
}
